lib/http-client.js

```diff
@@ -66,6 +69,33 @@ export default class HttpClient {
             timeout,
         });
 
+        this.#breakerCounter = this.#metrics.counter({
+            name: 'http_client_breaker_events',
+            description: 'Metrics on breaker events',
+        });
+        for (const eventName of this.#breaker.eventNames()) {
+            //@ts-ignore
+            const event = eventName;
+            //@ts-ignore
+            this.#breaker.on(event, (e) => {
+                let obj = Array.isArray(e) ? e[0] : e;
+                this.#breakerCounter.inc({
+                    labels: {
+                        //@ts-ignore
+                        name: event,
+                        ...(obj && obj.path && { path: obj.path }),
+                        ...(obj && obj.origin && { origin: obj.origin }),
+                    },
+                });
+            });
+        }
+        // @ts-ignore
+        this.#metrics.on('error', (error) => {
+            this.#logger.error(
```

Comment:
Do we want to log at the error level for this? I'm guessing if this errors we have other logs yelling at us 😅